spinn_front_end_common/abstract_models/impl/tdma_aware_application_vertex.py

```diff
@@ -44,12 +45,13 @@ class TDMAAwareApplicationVertex(ApplicationVertex):
         "The TDMA fell behind by {} times on core {}, {}, {}. "
         "try increasing the time_between_cores in the corresponding .cfg")
 
-    def __init__(self, label, constraints, max_atoms_per_core):
+    def __init__(self, label=None, constraints=None,
```

Comment:
I disagree with adding default values here!   That is the role of the super classes.

Reply:
Removed